metadata-builders/src/test/java/com/linkedin/metadata/builders/search/DatasetIndexBuilderTest.java

```diff
@@ -42,18 +41,20 @@ public void datasetPropertiesSetsDescription() {
     final DatasetProperties datasetProperties = new DatasetProperties().setDescription("baz");
     final DatasetSnapshot datasetSnapshot = ModelUtils.newSnapshot(DatasetSnapshot.class, datasetUrn,
         Collections.singletonList(ModelUtils.newAspectUnion(DatasetAspect.class, datasetProperties)));
-    final DatasetDocument expectedDocument = new DatasetDocument().setUrn(datasetUrn)
+    final DatasetDocument expectedDocument1 = new DatasetDocument().setUrn(datasetUrn).setDescription("baz");
+    final DatasetDocument expectedDocument2 = new DatasetDocument().setUrn(datasetUrn)
         .setBrowsePaths(new StringArray("/prod/foo/bar"))
         .setOrigin(FabricType.PROD)
         .setName("bar")
-        .setPlatform("foo")
-        .setDescription("baz");
+        .setPlatform("foo");
 
     // when
     final List<DatasetDocument> actualDocs = _indexBuilder.getDocumentsToUpdate(datasetSnapshot);
 
     // then
-    assertThat(actualDocs).containsExactly(expectedDocument);
+    assertEquals(actualDocs.size(), 2);
```

Comment:
Why turn back to TestNG?

Reply:
+1, keep this assertj

Reply:
Turned it back to TestNG to keep it consistent with tests of other search index builders.

Reply:
Any reason why assertj is preferred over TestNG?

Reply:
Changed to using assertj. We really should move other classes to using assertj framework instead of TestNG to avoid confusion.

Reply:
Agree, but it is non trivial (afaik there's no existing converter).

We can just use it in new tests / prefer it going forward.